# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.h
# Commit: ae4f46dfec58
# Full Hash: ae4f46dfec5886a64797be7073f63bda296c3541
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-29 20:54 +0000
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P1: Implement details relationship for CSS anchored local accessibles. r=Jamie,emilio
# ==============================================================================

diff -r 55449173b0e6 -r ae4f46dfec58 layout/base/PresShell.h
--- a/layout/base/PresShell.h	Mon Sep 29 20:39:23 2025 +0000
+++ b/layout/base/PresShell.h	Mon Sep 29 20:54:29 2025 +0000
@@ -777,6 +777,10 @@
     mAnchorPosPositioned.RemoveElement(aFrame);
   }
 
+  const nsTArray<nsIFrame*>& GetAnchorPosPositioned() const {
+    return mAnchorPosPositioned;
+  }
+
 #ifdef MOZ_REFLOW_PERF
   void DumpReflows();
   void CountReflows(const char* aName, nsIFrame* aFrame);
